kolla/node_custom_config/blazar.conf

```diff
@@ -47,6 +47,7 @@ heartbeat_in_pthread = false
 before_end = email
 email_relay = {{ blazar_email_relay }}
 enable_polling_monitor = true
+dry_polling_monitor = true
```

Comment:
This means that dry-run is enabled by default?
(optional) Maybe we can change the name of the flag to `enable_polling_monitor_dry_run`?

I would request that we set this via a variable, so that it can be set from defaults.yml, otherwise a site will need to use their own node_custom_config to override.

Something like:
```
enable_polling_monitor = {{ blazar_physical_polling_monitor }}
dry_polling_monitor = {{ blazar_physical_polling_monitor_dry_run }}
```